### PR TITLE
XAMARIN-43498-newly added feature is updated in xamarin scheduler

### PR DESCRIPTION
**Pull request by saiganesh-sakthivel**
XAMARIN-43498-newly added feature is updated in xamarin scheduler is reviewed by @Chrispineagunja

### CODE REVIEW

File: Xamarin/Scheduler/DayView.md [hunk 1]
```diff
@@ -437,6 +437,29 @@ schedule.DayViewSettings = dayViewSettings;
...
+### Customize time ruler label font appearance
+
+you can change the appearance of time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of `DayLabelSettings` property in `DayViewSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+<schedule:DayViewSettings.DayLabelSettings>
+    <schedule:DayLabelSettings TimeLabelFontAttributes="Bold">
+        <schedule:DayLabelSettings.TimeLabelFontFamily>
+            <OnPlatform x:TypeArguments="x:String" iOS="Lobster-Regular" Android="Lobster-Regular.ttf" WinPhone="Assets/Lobster-Regular.ttf#Lobster" />
```
Comment (Nijamudeen-Mohamed) [inline Xamarin/Scheduler/DayView.md:449]: Do we need winphone code?

File: Xamarin/Scheduler/DayView.md [hunk 2]
```diff
@@ -437,6 +437,29 @@ schedule.DayViewSettings = dayViewSettings;
 
 ![Schedule customizing time label appearance day view](daymodule_images/xamarin.forms-schedule-timelabel-appearance.png)
 
+### Customize time ruler label font appearance
```
Comment (Nijamudeen-Mohamed) [inline Xamarin/Scheduler/DayView.md:440]: Customize time ruler label font

File: Xamarin/Scheduler/WeekView.md [hunk 3]
```diff
@@ -448,6 +448,29 @@ schedule.WeekViewSettings = weekViewSettings;
 
 ![Time label appearance in schedule xamarin forms](weekview_images/xamarin.forms-schedule-timelabel-appearance.png)
 
+### Customize time ruler label font appearance
```
Comment (Nijamudeen-Mohamed) [inline Xamarin/Scheduler/WeekView.md:451]: Customize time ruler label font

File: Xamarin/Scheduler/DayView.md (hunk 1 above)
Comment (saiganesh-sakthivel) [reply Xamarin/Scheduler/DayView.md:449]: common code for the UWP platform

File: Xamarin/Scheduler/DayView.md [hunk 4]
```diff
@@ -437,6 +437,29 @@ schedule.DayViewSettings = dayViewSettings;
 
 ![Schedule customizing time label appearance day view](daymodule_images/xamarin.forms-schedule-timelabel-appearance.png)
 
+### Customize time ruler label font
+
+you can change the appearance of time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of `DayLabelSettings` property in `DayViewSettings`.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/DayView.md:442]: Change the appearance of the time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of the `DayLabelSettings` property in the `DayViewSettings.`

File: Xamarin/Scheduler/DayView.md [hunk 5]
```diff
@@ -437,6 +437,29 @@ schedule.DayViewSettings = dayViewSettings;
...
+</schedule:DayViewSettings.DayLabelSettings>
+{% endhighlight %}
+{% highlight c# %}
+Schedule.DayViewSettings.DayLabelSettings.TimeLabelFontAttributes = FontAttributes.Bold;
+Schedule.DayViewSettings.DayLabelSettings.TimeLabelFontFamily = Device.OnPlatform("Lobster-Regular", "Lobster-Regular.ttf", "Assets/Lobster-Regular.ttf#Lobster");
+{% endhighlight %}
+{% endtabs %}
+
+>**NOTE**
+FontAttributes and FontFamily are native to the platform. Custom font and the font which are not available in the specified platform will not be applied.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/DayView.md:461]: The FontAttributes and FontFamily are native to the platform. The custom font and a font that is not available in the specified platform will not be applied.

File: Xamarin/Scheduler/Timeline-view.md [hunk 6]
```diff
@@ -453,6 +453,29 @@ schedule.TimelineViewSettings = timelineViewSettings;
 
 ![Time label appearance in xamarin forms Timeline view](timeline-view-images/xamarin-forms-timeline-view-time-label-appearance.png)
 
+### Customize time ruler label font
+
+you can change the appearance of time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of `LabelSettings` property in `TimelineViewSettings`.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/Timeline-view.md:458]: Change the appearance of the time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of the `LabelSettings` property in the `TimelineViewSettings.`

File: Xamarin/Scheduler/Timeline-view.md [hunk 7]
```diff
@@ -453,6 +453,29 @@ schedule.TimelineViewSettings = timelineViewSettings;
...
+</schedule:TimelineViewSettings.LabelSettings>
+{% endhighlight %}
+{% highlight c# %}
+Schedule.WorkWeekViewSettings.LabelSettings.TimeLabelFontAttributes = FontAttributes.Bold;
+Schedule.WorkWeekViewSettings.LabelSettings.TimeLabelFontFamily = Device.OnPlatform("Lobster-Regular", "Lobster-Regular.ttf", "Assets/Lobster-Regular.ttf#Lobster");
+{% endhighlight %}
+{% endtabs %}
+
+>**NOTE**
+FontAttributes and FontFamily are native to the platform. Custom font and the font which are not available in the specified platform will not be applied.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/Timeline-view.md:477]: The FontAttributes and FontFamily are native to the platform. The custom font and the font which is not available in the specified platform will not be applied.

File: Xamarin/Scheduler/Timeline-view.md [hunk 8]
```diff
@@ -453,6 +453,29 @@ schedule.TimelineViewSettings = timelineViewSettings;
...
+
 ### Time Label size
 
 You can customize the size of the labels that mention the time by setting the [TimeLabelSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.TimelineLabelSettings.html#Syncfusion_SfSchedule_XForms_TimelineLabelSettings_TimeLabelSize) property of `LabelSettings` in `TimelineViewSettings`.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/Timeline-view.md:481]: Customize the size of the labels that mention the time by setting the [TimeLabelSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.TimelineLabelSettings.html#Syncfusion_SfSchedule_XForms_TimelineLabelSettings_TimeLabelSize) property of the `LabelSettings` in the `TimelineViewSettings.`

File: Xamarin/Scheduler/WeekView.md [hunk 9]
```diff
@@ -448,6 +448,29 @@ schedule.WeekViewSettings = weekViewSettings;
 
 ![Time label appearance in schedule xamarin forms](weekview_images/xamarin.forms-schedule-timelabel-appearance.png)
 
+### Customize time ruler label font
+
+you can change the appearance of time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of `WeekLabelSettings` property in `WeekViewSettings`.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/WeekView.md:453]: Change the appearance of the time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of the `WeekLabelSettings` property in the `WeekViewSettings.`

File: Xamarin/Scheduler/WeekView.md [hunk 10]
```diff
@@ -448,6 +448,29 @@ schedule.WeekViewSettings = weekViewSettings;
...
+</schedule:WeekViewSettings.WeekLabelSettings>
+{% endhighlight %}
+{% highlight c# %}
+Schedule.WeekViewSettings.WeekLabelSettings.TimeLabelFontAttributes = FontAttributes.Bold;
+Schedule.WeekViewSettings.WeekLabelSettings.TimeLabelFontFamily = Device.OnPlatform("Lobster-Regular", "Lobster-Regular.ttf", "Assets/Lobster-Regular.ttf#Lobster");
+{% endhighlight %}
+{% endtabs %}
+
+>**NOTE**
+FontAttributes and FontFamily are native to the platform. Custom font and the font which are not available in the specified platform will not be applied.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/WeekView.md:472]: The FontAttributes and FontFamily are native to the platform. The custom font and the font that is not available in the specified platform will not be applied.

File: Xamarin/Scheduler/WeekView.md [hunk 11]
```diff
@@ -448,6 +448,29 @@ schedule.WeekViewSettings = weekViewSettings;
...
+
 ## Time Label Size
 
 You can customize the size of the labels which are mentioning the time, by setting [TimeLabelSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.WeekLabelSettings.html#Syncfusion_SfSchedule_XForms_WeekLabelSettings_TimeLabelSize) property of `WeekLabelSettings` in `WeekViewSettings`.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/WeekView.md:476]: Customize the size of the labels which are mentioning the time by setting the [TimeLabelSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.WeekLabelSettings.html#Syncfusion_SfSchedule_XForms_WeekLabelSettings_TimeLabelSize) property of the `WeekLabelSettings` in the `WeekViewSettings.`

File: Xamarin/Scheduler/WorkWeekView.md [hunk 12]
```diff
@@ -463,6 +463,29 @@ schedule.WorkWeekViewSettings = workWeekViewSettings;
 
 ![Schedule customizing time label appearence work week view](workweekview_images/xamarin.forms-schedule-timelabel-appearance.png)
 
+### Customize time ruler label font
+
+you can change the appearance of time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of `WorkWeekLabelSettings` property in `WorkWeekViewSettings`.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/WorkWeekView.md:468]: Change the appearance of the time ruler label font family and font attribute by setting the `TimeLabelFontFamily` and `TimeLabelFontAttributes` property of the `WorkWeekLabelSettings` property in the `WorkWeekViewSettings.`

File: Xamarin/Scheduler/WorkWeekView.md [hunk 13]
```diff
@@ -463,6 +463,29 @@ schedule.WorkWeekViewSettings = workWeekViewSettings;
...
+</schedule:WorkWeekViewSettings.WorkWeekLabelSettings>
+{% endhighlight %}
+{% highlight c# %}
+Schedule.WorkWeekViewSettings.WorkWeekLabelSettings.TimeLabelFontAttributes = FontAttributes.Bold;
+Schedule.WorkWeekViewSettings.WorkWeekLabelSettings.TimeLabelFontFamily = Device.OnPlatform("Lobster-Regular", "Lobster-Regular.ttf", "Assets/Lobster-Regular.ttf#Lobster");
+{% endhighlight %}
+{% endtabs %}
+
+>**NOTE**
+FontAttributes and FontFamily are native to the platform. Custom font and the font which are not available in the specified platform will not be applied.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/WorkWeekView.md:487]: The FontAttributes and FontFamily are native to the platform. The custom font and the font that is not available in the specified platform will not be applied.

File: Xamarin/Scheduler/WorkWeekView.md [hunk 14]
```diff
@@ -463,6 +463,29 @@ schedule.WorkWeekViewSettings = workWeekViewSettings;
...
+
 ## Time Label Size
 
 You can customize the size of the labels which are mentioning the time, by setting [TimeLabelSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.WorkWeekLabelSettings.html#Syncfusion_SfSchedule_XForms_WorkWeekLabelSettings_TimeLabelSize) property of `WorkWeekLabelSettings` in `WorkWeekViewSettings`.
```
Comment (Chrispineagunja) [inline Xamarin/Scheduler/WorkWeekView.md:491]: Customize the size of the labels which are mentioning the time by setting the [TimeLabelSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.WorkWeekLabelSettings.html#Syncfusion_SfSchedule_XForms_WorkWeekLabelSettings_TimeLabelSize) property of the `WorkWeekLabelSettings` in the `WorkWeekViewSettings.`

**Comment by saiganesh-sakthivel** [comment]
rebuild

**Review by Jeyasri-Murugan — APPROVED** [review]

File: Xamarin/Scheduler/DayView.md (hunk 2 above)
Comment (saiganesh-sakthivel) [reply Xamarin/Scheduler/DayView.md]: modified the changes as discussed

File: Xamarin/Scheduler/WeekView.md (hunk 3 above)
Comment (saiganesh-sakthivel) [reply Xamarin/Scheduler/WeekView.md]: modified the changes as discussed

**Review by Nijamudeen-Mohamed — APPROVED** [review]
fine

**Comment by saiganesh-sakthivel** [comment]
rebuild

**Pull request merged**